src/components/PaginatedTable.js

```diff
@@ -72,9 +74,13 @@ function PaginatedTable ({
   };
 
   const onChangeTable = (pagination, filters, sorter) => {
-    const key = sorter.column?.dataIndex
+    const dataIndex = sorter.column?.dataIndex
+    const key = dataIndex ? dataIndexToString(dataIndex) : dataIndex
     const order = sorter.order
-    if (sortBy.key !== key || sortBy.order !== order)
+
+    const sortByKey = sortBy.key ? dataIndexToString(sortBy.key) : sortBy.key
```

Comment:
Why is `dataIndexToString` needed here? The store key should always be a string.